src/server/test/unit/liquidTagGrammer.test.ts

```diff
@@ -0,0 +1,330 @@
+  expect(parserOutput.get('31')).eq('key');
+  expect(parserOutput.get('45')).eq('name');
+  expect(parserOutput.get('56')).eq('id');
+
+});
+
+it('test valid index value(entitylist with attribute key,name, id) in Liquid Expression(Comma separated)', () => {
+  const parserOutput = parseLiquidAndGetKeyValue("{% include 'entity_list' key: 'list1',name: 'test',id: '123' %}");
+  expect(parserOutput.size).eq(3);
+  const liquidExpKeys = parserOutput.keys();
+  expect(liquidExpKeys.next().value).eq('31');
+  expect(liquidExpKeys.next().value).eq('45');
+  expect(liquidExpKeys.next().value).eq('56');
+  expect(parserOutput.get('31')).eq('key');
+  expect(parserOutput.get('45')).eq('name');
+  expect(parserOutput.get('56')).eq('id');
+});
+
+it('entitylist( value contains spaces but still the index is same )', () => {
+  const parserOutput = parseLiquidAndGetKeyValue("{% include 'entity_list' key: '     list1' %}");
+  expect(parserOutput.size).eq(1);
+  const liquidExpKeys = parserOutput.keys();
+  expect(liquidExpKeys.next().value).eq('31');
+  expect(parserOutput.get('31')).eq('key');
+});
+
+it('entitylist( value enclosed in double/single quotes return same parsed output)', () => {
+  const parserOutputWithDoubleQuotes = parseLiquidAndGetKeyValue(`{% include 'entity_list' key: "list1" %}`);
+  const parserOutputWithSingleQuotes = parseLiquidAndGetKeyValue("{% include 'entity_list' key: 'list1' %}");
+  const liquidExpKeysDouble = parserOutputWithDoubleQuotes.keys();
+  const liquidExpKeysSingle = parserOutputWithSingleQuotes.keys();
+  expect(liquidExpKeysDouble.next().value).eq('31');
+  expect(liquidExpKeysSingle.next().value).eq('31');
+  expect(parserOutputWithDoubleQuotes.get('31')).eq('key');
+  expect(parserOutputWithSingleQuotes.get('31')).eq('key');
+});
+
+
+ //#endregion 
+
+ //#region "Scenarios for Web Form"
+
+ it('test valid tag(webform) in Liquid Expression', () => {
+  const parsedTag = parseLiquidTag("{% webform name: 'test' %}");
+    expect(parsedTag).eq('webform');
+ });
+
+ it('test valid index value(webform with attribute name) in Liquid Expression', () => {
+     const parserOutput = parseLiquidAndGetKeyValue("{% webform name: 'test' %}");
+     expect(parserOutput.size).eq(1);
+     const liquidExpKeys = parserOutput.keys();
+     expect(liquidExpKeys.next().value).eq('18');
+     expect(parserOutput.get('18')).eq('name');
+  });
+
+  it('test valid index value(webform with attribute id) in Liquid Expression', () => {
+    const parserOutput = parseLiquidAndGetKeyValue("{% webform id: '1' %}");
+    expect(parserOutput.size).eq(1);
+    const liquidExpKeys = parserOutput.keys();
+    expect(liquidExpKeys.next().value).eq('16');
+    expect(parserOutput.get('16')).eq('id');
+ });
+
+  it('webform- grammar should be unambiguous and output length should be 1 only', () => {
+    const parserOutputLength = checkParsedOutputLength("{% webform name: 'test' %}");
+    expect(parserOutputLength).eq(1);
+ });
+
+
```

Comment:
nit: yeah, I know I'm nitpicking, please bear with me... as a human code reader, I do favor clean and consistent formatting :-)

Reply:
done